engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -162,16 +162,22 @@ private void processInput(EntityRef entity, CharacterMovementComponent character
         Quat4f viewRot;
         switch (characterMovementComponent.mode) {
             case WALKING:
-                viewRot = new Quat4f(TeraMath.DEG_TO_RAD * lookYaw, 0, 0);
-                viewRot.rotate(relMove, relMove);
+                if (!playerCamera.isTracked()) {
```

Comment:
Do I understand correctly that this is here so that it does not give extra information to the camera so that the VR stuff can hook into a lower system and provide this information?  If so (which I give myself only a 50% chance of being on the right track), could the playerCamera just ignore this data instead of making 2 different logic paths through these systems?